README.md

```diff
@@ -101,6 +101,15 @@ $ kubectl get no -w
 **Note:-** You should be seeing nodes joining the cluster within less than minutes.
 
 ---
+#### EKS cluster upgrade using new asg file in terraform
+Create a new eks-worker-node-v1.tf file with different name and below changes you have to do for EKS cluster upgrade.
+* Change the userdata name to new version(eks-worker-node-upgrade-v2.tf) and should not conflict with old one.
+* Change the Launch configuration and autoscalling group name to new version and should not conflict with old one.
+* Change the ami to which your going upgrade EKS version provided by AWS -- ##eks-worker-ami -- change to new version  
+* And in the new worker node file(eks-worker-node-upgrade-v2.tf), we have updated how to use taint for dedicated node.
+* Once you apply new tf file the new nodes will spin up and post that move the workload to the new one and delete old nodes.
+* Please reffer eks-worker-node-upgrade-v2.tf file to upgrade the EKS cluster for reference.
+
 
```

Comment:
@nandeeshb09 can we also add a one-line command on how to execute this via command-line arguments?

Reply:
Its just terraform apply...there are no command line arguments here..